2023-04-14 14:23:14,568 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Configure stats pid to 33216
2023-04-14 14:23:14,568 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Loading settings from /Users/emilywesel/.config/wandb/settings
2023-04-14 14:23:14,568 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Loading settings from /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/settings
2023-04-14 14:23:14,568 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-14 14:23:14,568 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'main.py'}
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:_log_setup():476] Logging user logs to /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/run-20230414_142314-5atunbs1/logs/debug.log
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/run-20230414_142314-5atunbs1/logs/debug-internal.log
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:init():516] calling init triggers
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:init():569] starting backend
2023-04-14 14:23:14,569 INFO    MainThread:33216 [wandb_init.py:init():573] setting up manager
2023-04-14 14:23:14,580 INFO    MainThread:33216 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-14 14:23:14,585 INFO    MainThread:33216 [wandb_init.py:init():580] backend started and connected
2023-04-14 14:23:14,596 INFO    MainThread:33216 [wandb_init.py:init():658] updated telemetry
2023-04-14 14:23:14,637 INFO    MainThread:33216 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-04-14 14:23:17,687 WARNING MsgRouterThr:33216 [router.py:message_loop():77] message_loop has been closed
2023-04-14 14:23:24,697 ERROR   MainThread:33216 [wandb_init.py:init():1105] error
Traceback (most recent call last):
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1078, in init
    run = wi.init()
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 697, in init
    result = handle.wait(
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 259, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
